_posts/2016-01-01-Markdown Reference.md

```diff
@@ -345,46 +352,46 @@ Use the `printf()` function.
 
 ### Strikethrough
 
-GFM adds syntax to create strikethrough text, which is missing from standard Markdown.
+GFM adds syntax to create strikethrough text, which is missing from standard Markdown. <!-- Comment: The same goes for tables, footnotes etc. but you've not mentioned that then. -->
```

Comment:
> Comment: The same goes for tables, footnotes etc. but you've not mentioned that then.

You could just mention them explicitly in the document, or make an extra issue to record this. But let's remove such "Comment" since users / developers may read it.

Reply:
OK, I wasn't sure of the best way to raise questions like this. 
I will add notes that tables and footnotes are also extensions.